README.md

```diff
@@ -101,14 +93,13 @@ CAUTION: Every time you re-deploy to Lambda using the `lambduh-maven-plugin`, yo
 manually re-add "Alexa Skills Kit" as the function's _Event Source_.  You do not need to
 do this if you deploy your code through the Lambda UI in AWS Console.
 
-### Authors
+## Authors

```

Comment:
@bbodenmiller Let's just remove the "Authors" section entirely.  We don't have this in any other OBA repo README.